tests/test_authentication_views.py

```diff
@@ -0,0 +1,58 @@
+"""
+Tests for the authentication views
+"""
+import pytest
+from django.test import Client
+from django.contrib.auth.models import User
+from authentication.models import UserBank
+
+
+@pytest.mark.django_db(transaction=True)
+def test_logout():
+    """
+    Test logout redirect
+    """
+    test_client = Client()
+    logout_test = test_client.get("/logout")
+    response_code = logout_test.status_code
+    assert response_code == 302
+    url = logout_test.url
+    assert url == "/login"
+
+
+@pytest.mark.django_db(transaction=True)
+def test_setup_bank():
+    """
+    Test bank setup page
+    """
+    test_client = Client()
+    user1 = User.objects.create(username='user', password="a")
+    test_client.force_login(user1)
+    not_set_up = test_client.get("/setup_bank")
+    response_code_1 = not_set_up.status_code
+    assert response_code_1 == 200
+    userbank = UserBank(
+        user=user1, item_id="hi",
+        access_token="Bye", institution_name="bankofcool",
+        current_balance_field=10, account_name_field="coolaccount",
+        income_field=30, expenditure_field=5
+    )
+    userbank.save()
+    set_up = test_client.get("/setup_bank")
+    response_code_2 = set_up.status_code
+    assert response_code_2 == 302
+    url = set_up.url
+    assert url == "/home"
```

Comment:
Do you need to mock plaid for this testcase?

Reply:
I don't think so? What I'm testing is just to see how the view reacts to when we have a userbank setup vs. when we don't and I set up the userbank manually. It shouldn't interact with/trigger plaid at all.